src/test/java/org/symphonyoss/symphony/messageml/elements/form/TextAreaTest.java

```diff
@@ -1,9 +1,10 @@
-package org.symphonyoss.symphony.messageml.elements;
+package org.symphonyoss.symphony.messageml.elements.form;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 
 import org.junit.Test;
+import org.symphonyoss.symphony.messageml.elements.*;
```

Comment:
Can we have only the needed imports? The wildcard one here is possibly not needed.